clients/fides-js/src/fides-tcf.ts

```diff
@@ -126,8 +128,6 @@ const init = async (config: FidesConfig) => {
     dispatchFidesEvent("FidesInitialized", cookie, config.options.debug);
   }
   dispatchFidesEvent("FidesUpdated", cookie, config.options.debug);
-
-  tcf();
```

Comment:
moving this farther up in the lifecycle is actually kind of important, otherwise the CMP API doesn't get initialized (since originally it wasn't called until after the `FidesInitialized` events). this resulted in a state where if you opened the IAB's chrome extension before opting in or out of anything, it wouldn't have any information, even though it should if we had previously agreed to something